test/Interop/Cxx/Inputs/IntBox.h

```diff
@@ -0,0 +1,3 @@
+struct IntBox {
```

Comment:
"IntWrapper"? I'd prefer to not use "box" because the int is not allocated on the heap.